cfgov/v1/migrations/0031_init_data.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.models import User
+from django.db import migrations, models
+from django.conf import settings
+
+from ..models.events import EventLandingPage
+from ..models.base import CFGOVPage
+
+from wagtail.wagtailcore.models import Page
+
+
+def initial_data(apps, schema_editor):
+    if settings.DEBUG:
+        admin_user = User(username='admin',
+                          password=make_password(os.environ.get('ADMIN_PW')),
+                          is_superuser=True, is_active=True, is_staff=True)
+        admin_user.save()
+
+        # Renaming Site root name to `V1`
+        site_root = Page.objects.get(id=2)
```

Comment:
@vccabral so `Page` is a wagtail model that is always migrated before any of our v1 migrations run because we are subclassing it for our own purposes. So i don't see any issues currently